docs/howto/outside_controller_use.md

```diff
@@ -0,0 +1,36 @@
+## Using AMS Outside Of A Controller
+
+### Serializing a resource 
+
+In the current version of AMS, serializing resources outside controllers is
+fairly simple:
+
+```ruby
+# Create our resource
+post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
+
+# Optional options parameters
+options = {}
+
+# Create the serializer
+serializer = ActiveModel::SerializableResource.serialize(post, options)
+```
+
+This will return an instance of the SerialzableResource class that will allow you to serialze your model. 
```

Comment:
As a reader, I'm left with an `ActiveModel::SerializableResource` although I was expecting information to make an actual json string.